acrobot.py

```diff
@@ -93,9 +109,10 @@ def solve_ilqr(solver, x0, u_guess):
 if method == "ilqr":
     # Set up optimizer
     num_steps = int(T/dt)
+    interpolation_method = utils_derivs_interpolation.derivs_interpolation(keypoint_method, minN, maxN, jerk_threshold, iterative_error_threshold)
```

Comment:
The cleanest thing might be to update this `interpolation_method` object to have `None` (or string `"none"`) as a possible keypoint method